assets/donate.css

```diff
@@ -0,0 +1,136 @@
+:root {
+  --gray: #798890;
+  --light-blue: #0f81e8;
+}
+
+.payments-titles {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    margin-bottom: 1em; /* 16px */
+  }
+
+  .payments-title {
+    border-bottom: 0.1875rem solid;  /* 3px */
+    border-bottom-color: #798890;
+    border-bottom-color: var(--gray);
+    width: 100%;
+    margin-right: 0.125rem; /* 2px */
+    text-align: center;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .payments-title-active {
+    border-bottom-color: #0f81e8;
```

Comment:
Could this also be initialized as a variable to keep things consistent?

Reply:
it is here: https://github.com/350org/ak_intl_v3/pull/308/files/628769dec562231255112095496cd1043cfde97b#diff-a0a5ba7ff20ec1dd96f860e068736e341ae87778d2c0cd386d34f3b3c01aebb3R28, I just included a fallback for the reason mentioned above